src/scss/components/topnav/index.scss

```diff
@@ -62,63 +73,58 @@
       font-size: 0.813rem;
       height: 3.5rem;
       line-height: 1;
-      padding: 0 0.563rem;
       text-align: center;
       text-decoration: none;
-      text-transform: uppercase;
+      display: flex;
```

Comment:
Is this value from the new specs?

Reply:
yes, there is a change how navigation icons to be displayed. Based on that change we need display: flex